### ISSUE TITLE
"Cannot read property 'short' of undefined" in resin flash

### ISSUE THREAD

**Issue by pimterry** [balena-io/balena-cli#589]
https://sentry.io/resinio/cli/issues/306413650/

```
TypeError: Cannot read property 'short' of undefined
  File "/Users/dreid/.nvm/versions/node/v8.0.0/lib/node_modules/resin-cli/node_modules/inquirer-dynamic-list/node_modules/inquirer/lib/prompts/list.js", line 94, in DynamicList.Prompt.render
    message += chalk.cyan( this.opt.choices.getChoice(this.selected).short );
  File "/Users/dreid/.nvm/versions/node/v8.0.0/lib/node_modules/resin-cli/node_modules/inquirer-dynamic-list/build/dynamic-list.js", line 155, in DynamicList.module.exports.DynamicList.render
    return DynamicList.__super__.render.apply(this, arguments);
  File "/Users/dreid/.nvm/versions/node/v8.0.0/lib/node_modules/resin-cli/node_modules/inquirer-dynamic-list/node_modules/inquirer/lib/prompts/list.js", line 116, in DynamicList.Prompt.onSubmit
    this.render();
  File "/Users/dreid/.nvm/versions/node/v8.0.0/lib/node_modules/resin-cli/node_modules/inquirer-dynamic-list/build/dynamic-list.js", line 130, in DynamicList.module.exports.DynamicList.onSubmit
    return DynamicList.__super__.onSubmit.apply(this, arguments);
  File "/Users/dreid/.nvm/versions/node/v8.0.0/lib/node_modules/resin-cli/node_modules/inquirer-dynamic-list/node_modules/rx-lite/rx.lite.js", line 1535, in AnonymousObserver.Rx.AnonymousObserver.AnonymousObserver.next
    this._onNext(value);
...
(36 additional frame(s) were not displayed)

TypeError: Cannot read property 'short' of undefined
```

**Comment by pdcastro** [comment]
Potentially affected (fixed?) by pull request #1052.

**Comment by pdcastro** [comment]
This error is no longer reported in sentry.io.
